### PR TITLE
My Pull Request

### PR DESCRIPTION
**Pull request by ChrisOwen101**
Change a word

### CODE REVIEW

**Comment by ChrisOwen101** [comment]
This is a good change!

File: README.md [hunk 1]
```diff
@@ -6,4 +6,4 @@ This is my name
 
 Hello
 
-Master
+Branch_New
```
Comment (ChrisOwen101) [inline README.md]: 👍

Comment (yusuf963) [reply README.md]: great Chris

Comment (Rayhan94) [reply README.md]: nice

**Pull request closed without merging**